Formula/kubefwd.rb

```diff
@@ -0,0 +1,17 @@
+class Kubefwd < Formula
+    desc "Bulk port forwarding Kubernetes services for local development"
+    homepage "https://github.com/txn2/kubefwd"
+    url "https://github.com/txn2/kubefwd.git"
+    version "1.9.2"
+    sha256 "3f49349f7de2c8f60ae4d1248be4f936c9082d2b23cf9fa093caef20a60685d4"
+  
+    depends_on "go@1.11" => :build
+  
+    def install
+      bin.install "kubefwd"
+    end
+  
+    test do
+      assert_natch "Kubefwd version: 1.9.2", shell_output("#{bin}/kubefwd -h 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.